js/index.js

```diff
@@ -1831,9 +1831,9 @@ getApiVersionPath = function(version) {
 	} else if (minorVersion < 25) {
 		return null;
 	} else if (minorVersion <= 26) {
-		return minorVersion;
+		return "";
```

Comment:
As the check is `apiVersionPath === null`, we can keep minorVersion here and "26" on the next return.

Reply:
Done.

I have fixed the success function too.

In the end I understood what the color "whitesmoke" means:

All dashboards with whitesmoke color are the dashboards with access for your user.
If you try to "join" a group in a blank dashboard, it will change to "whitesmoke" background